scss/functions/_get-color.scss

```diff
@@ -7,4 +7,12 @@
   // if they are valid, return the color
   // if they are not valid warn 'Invalid color type or level' and return null
   // null is a special value in Sass that means nothing, and it is used to represent the absence of a value, you need to return null if the type or level is not valid
+
+  // @if (map.has-key($colors, $type, $level)) { 
+  @if (map.has-key($colors, $type) and map.has-key(map.get($colors, $type), $level)) {
```

Comment:
ini sebaiknya diubah jadi `@if map.has-key($colors, $type) and map.has-key(map.get($colors, $type), $level)` tanda kurungnya dihapus 1 ya